summit_blood_samples/compose/production/summit-blood-samples.service

```diff
@@ -0,0 +1,19 @@
+[Unit]
+Description=SUMMIT Blood Samples Management
+After=network.target docker.service
+Requires=docker.service
+
+[Service]
+Type=oneshot
+RemainAfterExit=true
+
+User=root
+WorkingDirectory=/home/cceacoo/SUMMIT-blood-samples/summit_blood_samples
+
+ExecStart=/usr/local/bin/docker-compose -f production.yml up -d
```

Comment:
What is the reason behind the different flags used by **ExecStart** (yes `-d` no `--build`) & **ExecReload** (no `-d` yes `--build`)?

Reply:
The missing -d is a mistake; I added it in a later iteration (googling docker systemd gives several options!). The `--build` is so that an explicit reload rebuilds the containers, but a normal start doesn't.